src/ucs/datastruct/arbiter.h

```diff
@@ -113,7 +113,9 @@ typedef enum {
 #define UCS_ARBITER_GROUP_ARBITER_SET(_group, _arbiter) \
     (_group)->arbiter = (_arbiter)
 #define UCS_ARBITER_GROUP_ARBITER_CHECK(_group, _arbiter) \
-    ucs_assert((_group)->arbiter == (_arbiter))
+    ucs_assertv((_group)->arbiter == (_arbiter), \
```

Comment:
minor: ARBITER_CHECK macro may be moved out of scope ```#if ENABLE_ASSERT```

Reply:
why?

Reply:
there is used ucs_assertXXX macro which is disabled when ENABLE_ASSERT is empty

Reply:
ok, ic